graphql/database/users/getUserNotifications.js

```diff
@@ -16,12 +16,12 @@ const getUserNotifications = async (userContext, user) => {
     const notifFeature = await getUserFeature(
       userContext,
       user,
-      NOTIF_SFAC_JAN_2023
+      NOTIF_SFAC_FEB_2023
     )
     const enabled = notifFeature.variation !== 'None'
     notifications = [
       ...(enabled
-        ? [{ code: NOTIF_SFAC_JAN_2023, variation: notifFeature.variation }]
+        ? [{ code: NOTIF_SFAC_FEB_2023, variation: notifFeature.variation }]
```

Comment:
The variation won't be used in this notification because it's not an experiment. However, I realize I made a mistake when defining [the `variation` field in the GraphQL schema](https://github.com/gladly-team/tab/blob/master/graphql/data/schema.js#L710): I set it to non-null when it should be nullable.

In summary: you can leave it for now, but at some point change the the schema to allow that field to be null.